crates/prometheus-closure-metric/Cargo.toml

```diff
@@ -10,4 +10,4 @@ description = "Library for a Prometheus metric that computes its value by given
 [dependencies]
 anyhow.workspace = true
 prometheus.workspace = true
-protobuf = { version = "2.28", features = ["with-bytes"] }
+protobuf.workspace = true
```

Comment:
The policy is that we keep them in the crate if it's used only once, this seems to be the case here?

Reply:
protobuf is used here and iota-proxy.

Reply:
Oh, sorry about that, I messed up my search